src/algebra/ordered_group.lean

```diff
@@ -613,6 +613,18 @@ sub_lt_iff_lt_add'.trans (lt_add_iff_pos_left _)
 
 end ordered_comm_group
 
+namespace decidable_linear_ordered_comm_group
+variables [s : decidable_linear_ordered_comm_group α]
+include s
+
+instance : decidable_linear_ordered_cancel_comm_monoid α :=
+{ le_of_add_le_add_left := λ x y z, le_of_add_le_add_left,
+  add_left_cancel := λ x y z, add_left_cancel,
+  add_right_cancel := λ x y z, add_right_cancel,
+  ..s }
```

Comment:
why isn't this just `{..s}`?

Reply:
because the other fields don't exist in a group, but are provable, see e.g.
https://github.com/leanprover/lean/blob/ceacfa7445953cbc8860ddabc55407430a9ca5c3/library/init/algebra/ordered_group.lean#L203